docs/howto/blueprints.rst

```diff
@@ -0,0 +1,24 @@
+Use Blueprints
+--------------
+
+You can use blueprints to break up your tasks.
+
+Firstly use blueprints to create a task collection::
+
+    from procrastinate import Blueprint
+
+    bp = Blueprint()
+
+    @bp.task()
+    def mytask(argument, other_argument):
+        ...
+
+Then register blueprint with the app after you have created it::
+
+    from procrastinate import AiopgConnector, App
+
+    app = App(connector=AiopgConnector())
+
+    app.register_blueprint(bp)
```

Comment:
```suggestion
    app.register_blueprint(blueprint=blueprint)
```

Reply:
The API doesn't require a kwarg, and seeing the word blueprint 3 times in succession just makes me dizzy :dizzy_face: